docs/cli.rst

```diff
@@ -24,17 +24,32 @@ and providing a simple YAML configuration file:
 .. code-block:: yaml
 
     # foobar.yaml
-
     AMQP_URI: 'amqp://guest:guest@localhost'
-
     WEB_SERVER_ADDRESS: '0.0.0.0:8000'
-
     rpc_exchange: 'nameko-rpc'
-
     max_workers: 10
-
     parent_calls_tracked: 10
 
+    LOGGING:
```

Comment:
I have trimmed it down some. It assumes you know about about how `dictConfig` works but I think that was probably the case beforehand anyway.